packages/lesswrong/lib/collections/posts/views.ts

```diff
@@ -437,7 +437,9 @@ const stickiesIndexPrefix = {
 
 
 Posts.addView("magic", (terms: PostsViewTerms) => {
-  const selector = forumTypeSetting.get() === 'EAForum' ? filters.nonSticky : { isEvent: false };
+  const selector = forumTypeSetting.get() === 'EAForum'
+    ? { ...filters.nonSticky, ...filters.uncurated }
```

Comment:
This ends up hiding posts if they're curated but you've read them, even if they would have shown up otherwise. That's not great. It means that you don't see the best posts as much. I would probably revert this sadly, I'm not sure, but I think so.

The only other thing would be to see how hard it would be to only hide them from this list if the post is curated **&** read. But my guess, without looking at this very hard, is that that's too hard to be worth it.